components/device-mgt/org.wso2.carbon.device.mgt.api/src/main/java/org/wso2/carbon/device/mgt/jaxrs/service/api/GroupManagementService.java

```diff
@@ -672,8 +732,8 @@ Response addDevicesToGroup(@ApiParam(
                                        required = true)
                                @Valid List<DeviceIdentifier> deviceIdentifiers);
 
-    @Path("/id/{groupId}/devices")
-    @DELETE
+    @Path("/id/{groupId}/devices/remove")
```

Comment:
Any specific reason to convert DELETE method into POST in here? What was there is more restfull IMO